duktape/build.gradle

```diff
@@ -18,8 +18,15 @@ model {
     buildTypes {
       release {
         ndk {
-          CFlags.addAll(['-Os', '-fomit-frame-pointer'])
-          cppFlags.addAll(['-Os', '-fomit-frame-pointer'])
+          CFlags.addAll(['-Os', '-fomit-frame-pointer', '-DNDEBUG'])
+          cppFlags.addAll(['-Os', '-fomit-frame-pointer', '-DNDEBUG'])
+        }
+      }
+      debug {
```

Comment:
My gradle knowledge is pretty limited.  Does anyone know how I can get the unit tests to use a debug build of libduktape.so so we can hit assertions in the C++ code?